python/dolfinx/fem/petsc.py

```diff
@@ -939,12 +947,12 @@ def solve(self) -> _Function:
 
     @property
     def L(self) -> Form:
-        """The compiled linear form"""
+        """The compiled linear form `F`."""
         return self._L
 
     @property
     def a(self) -> Form:
-        """The compiled bilinear form"""
+        """The compiled bilinear form of `a`."""
```

Comment:
Odd docstring - more-or-less says "property `a` returns `a`". The docstring needs to make clear what the compiled bilinear form of `a` is.